Content.Server/CartridgeLoader/Cartridges/MessagesCartridgeComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.CartridgeLoader.Cartridges;
+using Content.Server.Radio.Components;
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+[RegisterComponent]
+public sealed partial class MessagesCartridgeComponent : Component
+{
+    /// <summary>
+    /// The component of the last contacted server
+    /// </summary>
+    [DataField]
+    public MessagesServerComponent? LastServer = null;
+
```

Comment:
You cannot store components in datafields, this will break serialization and it's also not clear why this is needed.